spec/syntax.md

```diff
@@ -704,3 +647,80 @@ s = 1*( SP / HTAB / CR / LF )
 The grammar is formally defined in [`message.abnf`](./message.abnf)
 using the ABNF notation,
 as specified by [RFC 5234](https://datatracker.ietf.org/doc/html/rfc5234).
+
+
+
+### Complex Messages
```

Comment:
Just delete this section: It's not meant to be here, and it's in the git history.